devenv/loadtest/render_test.js

```diff
@@ -1,33 +1,58 @@
-import { check, group } from 'k6';
+import { check, fail, group } from 'k6';
 import { createClient } from './modules/client.js';
+import {
+  createTestOrgIfNotExists,
+  upsertTestdataDatasource,
+  upsertTestdataDashboard,
+} from './modules/util.js';
 
 export let options = {
-  noCookiesReset: true
+  noCookiesReset: true,
+  thresholds: { checks: [ { threshold: 'rate=1', abortOnFail: true } ] },
 };
 
 let endpoint = __ENV.URL || 'http://localhost:3000';
 const client = createClient(endpoint);
+const dashboard = JSON.parse(open('fixtures/graph_panel.json'));
 
-export default () => {
-  group("render test", () => {
-    if (__ITER === 0) {
-      group("user authenticates thru ui with username and password", () => {
-        let res = client.ui.login('admin', 'admin');
+export const setup = () => {
+  let grafanaSession;
+  group("user authenticates thru ui with username and password", () => {
+    let res = client.ui.login('admin', 'admin');
 
-        check(res, {
-          'response status is 200': (r) => r.status === 200,
-        });
-      });
-    }
-
-    if (__ITER !== 0) {
-      group("render graph panel", () => {
-        const response = client.ui.render();
-        check(response, {
-          'response status is 200': (r) => r.status === 200,
-        });
+    check(res, {
+      'response status is 200': (r) => r.status === 200,
+    });
+  });
+
+  const orgId = createTestOrgIfNotExists(client);
+  client.withOrgId(orgId);
+  upsertTestdataDatasource(client, dashboard.panels[0].datasource);
+  upsertTestdataDashboard(client, dashboard);
```

Comment:
```suggestion
  upsertDashboard(client, dashboard);
```
If you agree with my suggestion above